checkST2110.js

```diff
@@ -597,54 +755,66 @@ const test_20_72_3 = (sdp, params) => {
       }
     }
   }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: Test ST2110-20 Section 7.2 Test 3 - Exactframerate is as specified');
+  }
   return errors;
 };
 
-const packingModes = [ '2110GPM', '2110BPM' ];
+const packingModes = ['2110GPM', '2110BPM'];
 
 // Test ST 2110-20 Section 7.2 Test 4 - Check packing mode as per spec.
 const test_20_72_4 = (sdp, params) => {
-  let [ mtParams, errors ] = extractMTParams(sdp, params);
-  for ( let stream of mtParams ) {
+  let [mtParams, errors] = extractMTParams(sdp, params);
+  for (let stream of mtParams) {
     if (typeof stream.PM !== 'undefined') {
       if (packingModes.indexOf(stream.PM) < 0) {
         errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, parameter 'PM' (packing mode) is not one of the defined values, as per SMPTE ST 2110-20 Sections 7.2 and 6.3.`));
       }
     }
   }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: Test ST2110-20 Section 7.2 Test 4 - Packing mode is as per spec.');
+  }
   return errors;
 };
 
-// Test ST 2110-20 Section 7.2 Test 5 - Check SSN is the required fixed value
+// Test ST 2110-20 Section 7.2 or ST  Test 5 - Check SSN is the required fixed value
 const test_20_72_5 = (sdp, params) => {
-  let [ mtParams, errors ] = extractMTParams(sdp, params);
-  for ( let stream of mtParams ) {
+  let [mtParams, errors] = extractMTParams(sdp, params);
+  for (let stream of mtParams) {
     if (typeof stream.SSN !== 'undefined') {
       if (stream.SSN !== 'ST2110-20:2017') {
         errors.push(new Error(`Line ${stream._line}: For stream ${stream._streamNumber}, format parameter 'SSN' is not set to the required value 'ST2110-20:2017', as per SMPTE ST 2110-20 Section 7.2.`));
       }
     }
   }
+  if (params.verbose && errors.length == 0) {
+    console.log('Test Passed: Test ST2110-20 Section 7.2 Test 5 - SSN is the required fixed value \'ST2110-20:2017\'');
```

Comment:
Let's note a follow-up issue to do the fix up for the 2022 revisions of -10/-20/-21/-40...